python-sdk/tests/sql/operators/test_merge.py

```diff
@@ -198,7 +198,7 @@ def test_merge_with_the_same_schema(database_table_fixture, multiple_tables_fixt
             {"id": 4, "name": "Czwarte imię"},
         ]
     )
-
+    assert first_table.row_count == 4
```

Comment:
@rajaths010494 in this particular test, we are comparing if the values match the expected values (so the content of all rows and columns) - `test_utils.assert_dataframes_are_equal`. Which benefit do you think we gain by adding this additional assert?

Reply:
this is to check in first table in the database for the merge operation has the data. As you mentioned we export the table and validate it with dataframe.